src/libraries/System.DirectoryServices.Protocols/src/System/DirectoryServices/Protocols/ldap/LdapSessionOptions.cs

```diff
@@ -1093,7 +1093,6 @@ private static bool AddLdapHandleRef(LdapConnection ldapConnection)
             bool success = false;
             if (ldapConnection != null && ldapConnection._ldapHandle != null && !ldapConnection._ldapHandle.IsInvalid)
             {
-                RuntimeHelpers.PrepareConstrainedRegions();
```

Comment:
This project targets netstandard + netframework.
(This comment applies file-wide.)

Reply:
The file is in an ItemGroup that does not target netstandard.

My understanding of the `_$(NetFrameworkCurrent)` target framework is that is used for NuGet packaging so that the package can target netframework, but the code from this project will not actually be compiled or run when targeting netframework, as the GAC will already have this assembly under the same identity.

With that rationale, this file is only compiled for NetCoreApp.